packages/components/checkbox/src/Checkbox.tsx

```diff
@@ -20,41 +21,50 @@ export const Checkbox = forwardRef<HTMLButtonElement, CheckboxProps>(
       disabled,
       onCheckedChange,
       children,
-      ...others
+      ...rest
     },
     forwardedRef
   ) => {
+    const checkboxId = useId()
```

Comment:
IMO there is no need of using `checkbox` prefix is redundant since the component is named `Checkbox`